From 5e37f9cf53e196459c59faceb9317e4e755df4eb Mon Sep 17 00:00:00 2001
From: Jim Stichnoth <jstichnoth@mythtv.org>
Date: Mon, 28 Jul 2014 09:36:58 -0700
Subject: [PATCH] Captions: Delete *all* cc708 windows in
 CC708Reader::Reset().

Refs #12227.

Signed-off-by: Jim Stichnoth <jstichnoth@mythtv.org>
(cherry picked from commit e97f4f59806a32421a2d2ab4c79549286c56b6ab)

Conflicts:

	mythtv/libs/libmythtv/cc708reader.cpp
---
 mythtv/libs/libmythtv/cc708reader.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/cc708reader.cpp b/mythtv/libs/libmythtv/cc708reader.cpp
index 877930d..8f645f0 100644
--- a/mythtv/libs/libmythtv/cc708reader.cpp
+++ b/mythtv/libs/libmythtv/cc708reader.cpp
@@ -276,7 +276,7 @@ void CC708Reader::Reset(uint service_num)
 {
     CHECKENABLED;
     LOG(VB_VBI, LOG_INFO, LOC + QString("Reset(%1)").arg(service_num));
-    DeleteWindows(service_num, 0x7);
+    DeleteWindows(service_num, 0xff);
     DelayCancel(service_num);
 }
 
-- 
1.7.10.2

